app/controllers/admin/export/base_controller.rb

```diff
@@ -15,4 +15,8 @@ def forbidden!
   def not_found!
     respond_with Hash.new, status: :not_found
   end
+
+  def no_content
+    head :no_content
+  end
```

Comment:
I'd have thought you don't really need to dry this up as someone reading this then has to look this method up to understand quite what no_content means which seems a bit unnecessary for a 6 character saving. 

Doing a bit of Googling you might not actually need to specify head :no_content as Rails seems to have started doing that by default since Rails 5: https://github.com/rails/rails/pull/19377